packages/mui-base/src/Popover/Trigger/usePopoverTrigger.ts

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+import type {
+  UsePopoverTriggerParameters,
+  UsePopoverTriggerReturnValue,
+} from './usePopoverTrigger.types';
+
+/**
+ *
+ * API:
+ *
+ * - [usePopoverTrigger API](https://mui.com/base-ui/api/use-popover-trigger/)
+ */
+export function usePopoverTrigger(
+  params: UsePopoverTriggerParameters,
+): UsePopoverTriggerReturnValue {
+  const { getProps } = params;
+
+  const getTriggerProps = React.useCallback(
+    (externalProps = {}) => {
+      return getProps(externalProps);
```

Comment:
There is no point in having such an empty hook. I was thinking about something along these lines: https://github.com/mui/base-ui/blob/master/packages/mui-base/src/Dialog/Trigger/useDialogTrigger.ts

Reply:
The props for the trigger are already created within the root from `useInteractions`. It's pointless indeed (at least at the moment) and since it's not public I can remove it here, just created it for consistency reasons.

Reply:
Having the props defined in usePopoverTrigger will allow us to reuse the hook across Popover, Dialog, and others, making the overall bundle size smaller. If it requires much more code (or more complex code), we can use the root hook, though.